### DISCUSSION TITLE
conduction of PBPK model

### DISCUSSION THREAD

**Discussion by ZML0113** [Open-Systems-Pharmacology/Forum#1368] [Questions & Problems]
I tried to change the drug and physiology parameters many times, but the model still could not be verified well. Could you give me some constructive suggestion and opinion?

**Comment by StephanSchaller** [comment]
We need more info on the model (Model file?) the outcome (paste result figure?) to be able to help.

**Reply by ZML0113** [reply]
[1028.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9886526/1028.zip)
The model file is attached in the letter, and the oral dose of Tmax is much smaller than observed letter. I tried to change the parameters such as: Vmax of CYP3A4, Log p of tacrolimus, specific of intestinal permeability and specific organ permeability in the distribution part. But it still couldn't be verified well.
